robottelo/vm.py

```diff
@@ -257,6 +257,82 @@ def run(self, cmd):
 
         return ssh.command(cmd, hostname=self.ip_addr)
 
+    def fetch_hostname(self):
+        """Fetches short hostname from client and the domain from provisioning
+        server.
+
+        :return: Returns the FQDN of the host.
+
+        """
+        # NOTE: We will be using 'hostname -s' as currently the hostname being
+        # returned from virtual machine is short hostname. It is being done to
+        # handle the scenario where in the vm/client might return FQDN for
+        # a different provisioning server.
+        host = self.run('hostname -s')
+        domain = ssh.command(
+            u'hostname -d',
+            hostname=self.provisioning_server
+        )
+        return host.stdout[0] + u'.' + domain.stdout[0]
+
+    def configure_rhel_repo(self, rhel_repo):
+        """Configures specified Red Hat repository on the virtual machine.
+
+        :param rhel_repo: Red Hat repository link from properties file.
+        :return: None.
+
+        """
+        # 'Access Insights', 'puppet' requires RHEL 6/7 repo and it is not
+        # possible to sync the repo during the tests as they are huge(in GB's)
+        # hence this adds a file in /etc/yum.repos.d/rhel6/7.repo
+        repo_file = (
```

Comment:
@kbidarkar instead of adding a repo file on the fly, I think we can add:
```
self.run('wget -o /etc/yum.repos.d/rhel.repo {0}'.format(rhel_repo))
```
And pass the repo link to the rhel_repo in the clients['rhel*_repo'].
What is your opinion on that?